### ISSUE TITLE
Gameplay Ideas

### ISSUE THREAD

**Issue by 0x48piraj** [0x48piraj/Switch#4]
1. Time limit
2. And like Mario, set life to 3, 
3. Add fruits to the map, that can increase life, time
4. Introduce magic doors, some will lead to deaths, respawn, or gold.

**Comment by felizj17** [comment]
are these all one issue or are you trying to come up with new ideas, because I would like to work on this.

**Comment by 0x48piraj** [comment]
Hey @felizj17

Yes, I came up with some ideas, you can work on any of them or pitch new ones and make a Pull Request.

**Comment by felizj17** [comment]
Awesome I'll start work on these now and think about new ideas as I get a better idea of the game

**Comment by inglorious-ratbastard** [comment]
I've submitted a PR with some changes made reflecting your wish to countdown player lives from 3, implementing a game over refresh upon losing. Please review #5 submission for Hacktoberfest.

**Comment by felizj17** [comment]
I added a timer and adjusted the death counter into 3 life limit, with this implemented other things should be added such as a pause button that would stop the ai bots and timer as well as maybe a score; with which lives would be added after reaching certain benchmarks.

**Comment by felizj17** [comment]
Hello, is anyone there? Hope everything is ok @0x48piraj

**Comment by 0x48piraj** [comment]
Yes everything's fine, I'm sorry for responding late. Just commented on your Pull Request.
